modules/fox-spring/src/test/resources/com/ensolvers/fox/spring/pointcutAutowiring-context.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xmlns:context="http://www.springframework.org/schema/context"
+   xsi:schemaLocation="http://www.springframework.org/schema/beans      http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+                       http://www.springframework.org/schema/context    http://www.springframework.org/schema/context/spring-context-3.0.xsd
+                       ">
+
+   <context:annotation-config />
+   <context:component-scan base-package="com.ensolvers.fox.spring.aop" />
+</beans>
```

Comment:
Why are we using XML here? We need to use annotation-based configurations as we do in other projects as well

Reply:
Annotated examples will come as we gain experience.
XML is the first configuration kind Spring searches and is useful to provide simple or complex isolated contexts